README.md

```diff
@@ -6,4 +6,13 @@ This is the Dockerfile (etc.) used for building the image that runs python scrip
 
 Because the build process of this image has become quite lengthy, we've split it into three. The base images are at [here](https://github.com/Kaggle/docker-python0) and [here](https://github.com/Kaggle/docker-python1).
 
+**To start container [Learn more](http://blog.kaggle.com/2016/02/05/how-to-get-started-with-data-science-in-containers/comment-page-1/)
```

Comment:
I'd prefer to drop the double asterisk at the start, and have something like `For a guide to using the container, see [our blog](http://blog.kaggle.com/2016/02/05/how-to-get-started-with-data-science-in-containers).`